tests/samples/matmul_fill_static.mlir

```diff
@@ -1,9 +1,23 @@
-func.func @matmul_static(%lhs : tensor<128x256xf32>,
-    %rhs : tensor<256x512xf32>) -> tensor<128x512xf32> {
-  %empty = tensor.empty() : tensor<128x512xf32>
-  %cst = arith.constant 0.0 : f32
-  %fill = linalg.fill ins(%cst : f32) outs(%empty : tensor<128x512xf32>) -> tensor<128x512xf32>
-  %2 = linalg.matmul ins(%lhs, %rhs : tensor<128x256xf32>, tensor<256x512xf32>)
-      outs(%fill : tensor<128x512xf32>) -> tensor<128x512xf32>
-  return %2 : tensor<128x512xf32>
+// Example using a transform dialect script.
+
+// RUN: iree-opt %s   --iree-hal-target-backends=amd-aie \
```

Comment:
You can use `iree-compile` to make a more succinct command line https://github.com/nod-ai/iree-amd-aie/blob/a42ddaca69c1068f242c08cd204232f3078b2040/tests/samples/matmul_fill_static_i32.mlir#L1

Reply:
I liked this approach because 

1) it passes with different tensor sizes, as it does not lower to mlir-aie (for example changing to M=1024 results in a failure lowering to mlir-aie). 

2) I wanted a way to see the output of running the pass with transform dialect script, before lowering all the down to mlir-aie. 

Neither of these is a good reason to use this approach in a test, so I will change to the `iree-compile` approach. 

I guess I can use --mlir-print-ir-after=. with `iree-compile` to see the IR after transform dialect has run. 

@MaheshRavishankar - do you know if there is a way to see the IR after each of the individual passes described in the transform IR? i.e. after each of the `tile_using_forall`, `pad`, `bufferize_to_allocation` transform ops have modified the IR?